cfgov/cfgov/settings/base.py

```diff
@@ -700,9 +700,6 @@
     # Test financial well-being hub pages on Beta
     'FINANCIAL_WELLBEING_HUB': [('environment is', 'beta')],
 
-    # Home Page automatic Latest Updates
```

Comment:
Is this a feature flag that is no longer needed?

Reply:
Correct.